pyproject.toml

```diff
@@ -43,6 +43,8 @@ importlib_metadata = { version = "^1.1", python = "<3.8" }
 werkzeug = ">0.16.0"
 junit-xml = "^1.9"
 starlette = "^0.13.2"
+yarl = "^1.6.0"
+responses = "^0.12.0"
```

Comment:
I don’t see this dependency used :) seems like it could be removed

Reply:
Thanks - fixed in next patch